curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/understand-the-hazards-of-using-imperative-code.english.md

```diff
@@ -90,6 +99,41 @@ alert(finalTabs.tabs);
 <section id='solution'>
 
 ```js
```

Comment:
@RandellDawson Shouldn't this be:
````
```html
````
?

Reply:
@thecodingaviator This challenge is pure js, so I think it's fine as is.

Reply:
You're right, my bad